minigotchi-ESP32/display.h

```diff
@@ -1,29 +1,33 @@
-/**
- * display.h: header files for display.cpp
- */
-
 #ifndef DISPLAY_H
 #define DISPLAY_H
 
-#include "config.h"
-#include <Adafruit_GFX.h>
```

Comment:
can we keep these for the ssd1306, I don't want to remove these

```cpp
#include <Adafruit_GFX.h>
#include <SPI.h>
```

Reply:
Added back.